CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
+
+## v2.2.2
```

Comment:
Nitpick: `generate_collection_umm_var` is one of the end-user entry points of our code, so I think this change counts as a backwards compatible API change. I think that means this should be a minor version number change (so 2.3.0 rather than 2.2.2).

Reply:
ok